packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -158,7 +158,7 @@ export class AtomicSearchBox {
   /**
    * Event that is emitted when a standalone search box redirection is triggered. By default, the search box will directly change the URL and redirect accordingly, so if you want to handle the redirection differently, use this event.
```

Comment:
Actually, I notice that we don't generate docs for events currently. I'll add that to the list :loading: